pyEpiabm/pyEpiabm/intervention/case_isolation.py

```diff
@@ -21,12 +21,14 @@ def __init__(
         isolation_duration,
         isolation_probability,
         isolation_delay,
+        criterion,
```

Comment:
I don't find this name very obvious - can we rename it to something more descriptive? Happy to discuss ideas on wednesday, but can't give suggestions as I'm not actually sure what it is doing!

Reply:
Sorry I had meant to change this. This parameter should be 0 if symptoms are used to isolate people or 1 is testing should be used - I have changed the name to use_testing.